index.html

```diff
@@ -10,7 +10,7 @@
     <main>
       <div class="page">
         <div class="head">
-          <div class="face">:|</div>
+          <div class="face">:)</div>
```

Comment:
```suggestion
          <div class="face">:D</div>
```

haz que sonría más :D